src/main/resources/com/google/api/codegen/java/grpc_test.snip

```diff
@@ -39,7 +39,7 @@
     }
 
     @@After
-    public void tearDown() throws Exception {
+    public void tearDown() {
```

Comment:
These `throws Exception` clauses don't harm anyone, I think we should leave them. (JUnit is the only layer concerned really)

Reply:
OK, I will revert.